NDB.Covid19/NDB.Covid19/Locales/en.json

```diff
@@ -73,29 +72,27 @@
+  "SETTINGS_PAGE_2_CONTENT_TEXT": "\"The Smittestopp contact tracing app is a digital initiative in the fight against the spread of COVID-19.<br><br><b>Active and inactive status</b><br>When the app is active, you contribute to stopping the spread of COVID-19 in Norway and a number of EU and EEA countries. You can always turn registration on and off in the app by pressing the circular button on the home screen. When the app is inactive — as shown by a red circular button — your phone will stop making registrations.<br><br>When you have given your consent and allowed Bluetooth, the app will operate continuously. You can withdraw your consent under 'Processing of personal data'.<br><br><b>Sharing your test result</b><br>If you have tested positive, you can voluntarily share your status in the app by entering the home screen and pressing ‘Did you test positive?’. Other app users cannot see who you are. You must log in via the ID-porten.<br><br>After you have shared your status, other users will receive a message saying that they have been nearby a person infected with COVID-19.<br><br>You can see messages in the app on the home screen. Here you can also see if you have received a new exposure message.<br><br>If Smittestopp is inactive, you cannot receive messages. You can log in and share your status, but as there is no exchange of IDs when the app is inactive, there will be nobody to share your status with. <br><br><b>The technology</b> <br>The app uses Bluetooth and is based on built-in technology developed by Apple and Google. It comprises a system for notification of exposures. You can turn this on or off in your phone’s settings. In your phone’s settings, search for ‘covid’ or ‘smittestopp’ to turn the logging on or off. <br><br>Your phone generates a random Bluetooth ID every 15 minutes. When your phone is close to another app user’s phone, the two phones will exchange IDs.<br><br> Read more about the Smittestopp app at <a href=\\\\\"www.helsenorge.no/en/smittestopp\\\\\">www.helsenorge.no/en/smittestopp</a><br><br>\"",
```

Comment:
why so many backslashes in the link?

Reply:
Hmm... don't know. It comes straight from the excel sheet @nc-kano

Reply:
so it should be corrected there. it looks like copied text from somewhere and some tool has added additional symbols. also "\ at the beginning of this field seems to be not needed